### PR TITLE
$overwrite was equal to null even when using --overwrite. Fixed

### PR DESCRIPTION
**Pull request by huglester**
Fixed $overwrite was equal to null even when using --overwrite option
also small typo fix

### CODE REVIEW

File: classes/generate.php [hunk 1]
```diff
@@ -67,7 +67,7 @@ class Generate
 			}
 		}
 
-		$overwrite = \Cli::option('o') or \Cli::option('overwrite');
+		$overwrite = (\Cli::option('o') or \Cli::option('overwrite')) ? true : false;
```
Comment (philsturgeon) [inline classes/generate.php:70]: Please remove `? true : false` as that is already inferred.

**Comment by crynobone** [comment]
@huglester I guess what @philsturgeon meant was it should be just with ()

```
$overwrite = (\Cli::option('o') or \Cli::option('overwrite'));
```

This pull request should be able to fix #56

**Comment by philsturgeon** [comment]
Yep.

**Comment by huglester** [comment]
ah. didn't understood. somehow was telling me on IRC that this was working .. but for me it wasn't....
Thanks crybone for clarifying :)

**Comment by okhayat** [comment]
Just tested it. The issue is still on:

$ php oil g config date
Created config: APPPATH/config/date.php
$ php oil g config date
Error: APPPATH/config/date.php already exist, please use -overwrite option to force update
$ php oil g config date -o
Created config: APPPATH/config/date.php
$ php oil g config date --overwrite
Error: APPPATH/config/date.php already exist, please use -overwrite option to force update

The error message should also use two dashes for the --overwrite

**Comment by crynobone** [comment]
It works for me (with `$overwrite = (\Cli::option('o') or \Cli::option('overwrite'));`)

```
$ php oil g config date
Created config: APPPATH/config/date.php

$ php oil g config date
Error: APPPATH/config/date.php already exist, please use -overwrite option to force update

$ php oil g config date -o
Created config: APPPATH/config/date.php

$ php oil g config date --overwrite
Created config: APPPATH/config/date.php
```

**Comment by huglester** [comment]
For me it also works with this fix. I have earlier provided more cleaner fix suggested by @philsturgeon
https://github.com/fuel/oil/pull/58

**Comment by okhayat** [comment]
I've tested again with your fix, and it now works fine for me :) Closed the other issue.

**Pull request closed without merging**